src/main/java/com/sublinks/sublinksapi/federation/enums/ActorType.java

```diff
@@ -0,0 +1,16 @@
+package com.sublinks.sublinksapi.federation.enums;
+
+public enum ActorType {
+  USER("user"),
+  COMMUNITY("community");
```

Comment:
I'm using Person and Group to match activity pub types (haven't pushed that code yet). However, I'm open to discussion on if we stick with user/community instead

Reply:
I take it back, I did push some code on this. I started created JSON Schema files: https://github.com/sublinks/sublinks-federation/pull/43/files#diff-5d252b8ffdf59c8a7c138bfbb9b0d43075698195dc3cadba0c295641c387be2a